package.json

```diff
@@ -36,7 +36,7 @@
     "fuse.js": "^2.2.0",
     "glob": "^5.0.14",
     "highlight.js": "^8.9.1",
-    "linkifyjs": "2.0.0-beta.4",
+    "linkifyjs": "2.1.0",
```

Comment:
It does, I'll update and revert this change. The actual fix here is to modify the states and transitions used, so that is all that's required.